.nvmrc

```diff
@@ -0,0 +1 @@
+12.18
```

Comment:
Should this be `12.18.3`?

Reply:
I thought allowing some freedom in the patch version would be fine.